doc/man7/EVP_KDF-KB.pod

```diff
@@ -23,21 +23,34 @@ The supported parameters are:
 
 =item B<OSSL_KDF_PARAM_PROPERTIES> ("properties") <UTF8 string>
 
-=item B<OSSL_KDF_PARAM_DIGEST> ("digest") <UTF8 string>
+=item B<OSSL_KDF_PARAM_MODE> ("mode") <UTF8 string>
```

Comment:
The format of the parameter names and types was changed recently.  These should all be of the form:
```
=item "mode" (B<OSSL_KDF_PARAM_MODE>) <UTF8 string>
```